### PR TITLE
aarch + ppc + osx builds

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml
Comment (ngam) [inline conda-forge.yml:2]: isn't this supposed to be default instead of azure?

File: conda-forge.yml
Comment (ngam) [inline conda-forge.yml:3]: could you also add osx (both flavors)? thanks!
```suggestion
test_on_native_only: true
build_platform:
   osx_arm64: osx_64
 test_on_native_only: true
```

File: recipe/meta.yaml
Comment (ngam) [inline recipe/meta.yaml]:
```suggestion
  skip: True  # [not unix]
```

File: conda-forge.yml
Comment (ngam) [reply conda-forge.yml:2]: oh maybe they changed it since travis has been doing badly recently...

Comment (Tobias-Fischer) [reply conda-forge.yml:2]: aarch runs on azure by default, and I hate travis for ppc as it breaks every other day. But happy to change if you prefer.

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml
Comment (ngam) [reply conda-forge.yml:2]: I am not the maintainer, so my preference shouldn't matter :D 

I also hate travis!!! I am gonna copy this trick to other feedstocks

Comment (ngam) [reply conda-forge.yml:2]: THANK YOU, hopefully the osx runs pass too

**Comment by Tobias-Fischer** [comment]
@notestaff @ngam - this is ready for review / merge.

**Review by ngam — APPROVED** [review]

**Comment by ngam** [comment]
@Tobias-Fischer maybe you should merge this (if you're part of core) otherwise maybe ping the core team...

**Comment by Tobias-Fischer** [comment]
@conda-forge/core - it would be great if you could merge this. The maintainer hasn't responded for 10 days. Thanks!

**Pull request merged**
